CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ This will prevent BigQuery from throwing an error since non-partitioned tables c
 - Add stale messaging workflow to Github Actions [#103](https://github.com/dbt-labs/dbt-bigquery/pull/103)
 - Add unique_key to go in parity with unique_key as a list chagnes made in core [#119](https://github.com/dbt-labs/dbt-bigquery/pull/119/files)
 - Adding new Enviornment variable for integration testing puproses [#116](https://github.com/dbt-labs/dbt-bigquery/pull/116)
+- Convert unique id as a list tests to new pytest format ([#150](https://github.com/dbt-labs/dbt-bigquery/pull/150))
```

Comment:
I am also okay with not creating changelog for this kind of test conversion, less headach when cherry-picking since changie is not enabled here

Reply:
is this something we want to do across the board for adapters?

Reply:
My general rule of thumb has been, if something is not a change to the actual installed code (in this case, the `dbt-bigquery` Python package), we probably don't need a changelog entry for it. There are some exceptions, e.g. version-specific helper code for installation methods

Reply:
will remove them and re push up, thanks for the insight @ChenyuLInx and @jtcohen6